src/vm/tests/costs.rs

```diff
@@ -651,68 +666,52 @@ fn test_tracked_costs(prog: &str) -> ExecutionCost {
     let trait_contract_id =
         QualifiedContractIdentifier::new(p1_principal.clone(), "contract-trait".into());
 
-    let marf_kv = MarfedKV::temporary();
-    let mut clarity_instance = ClarityInstance::new(false, marf_kv);
-    clarity_instance
-        .begin_test_genesis_block(
-            &StacksBlockId::sentinel(),
-            &StacksBlockHeader::make_index_block_hash(
-                &FIRST_BURNCHAIN_CONSENSUS_HASH,
-                &FIRST_STACKS_BLOCK_HASH,
-            ),
-            &TEST_HEADER_DB,
-            &TEST_BURN_STATE_DB,
-        )
-        .commit_block();
-
-    let mut marf_kv = clarity_instance.destroy();
-
-    let mut store = marf_kv.begin(
-        &StacksBlockHeader::make_index_block_hash(
-            &FIRST_BURNCHAIN_CONSENSUS_HASH,
-            &FIRST_STACKS_BLOCK_HASH,
-        ),
-        &StacksBlockId([1 as u8; 32]),
-    );
+    with_owned_env(epoch, |mut owned_env| {
+        owned_env
+            .initialize_contract(trait_contract_id.clone(), contract_trait)
+            .unwrap();
+        owned_env
+            .initialize_contract(other_contract_id.clone(), contract_other)
+            .unwrap();
+        owned_env
+            .initialize_contract(self_contract_id.clone(), &contract_self)
+            .unwrap();
 
-    let mut owned_env = OwnedEnvironment::new_max_limit(
-        store.as_clarity_db(&TEST_HEADER_DB, &TEST_BURN_STATE_DB),
-        StacksEpochId::Epoch2_05,
-    );
+        let target_contract = Value::from(PrincipalData::Contract(other_contract_id.clone()));
 
-    owned_env
-        .initialize_contract(trait_contract_id.clone(), contract_trait)
-        .unwrap();
-    owned_env
-        .initialize_contract(other_contract_id.clone(), contract_other)
-        .unwrap();
-    owned_env
-        .initialize_contract(self_contract_id.clone(), &contract_self)
+        eprintln!("{}", &contract_self);
```

Comment:
do you mean to check these `eprintln` lines in? just checking

Reply:
Yeah -- `eprintln`s are useful in test code (which this line is in a tests module), because they get captured by the test runner and only display if the test fails.